frontend/public/locales/en/network-policy.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Name": "Name",
+  "Namespace": "Namespace",
+  "Pod selector": "Pod selector",
+  "Network policies": "Network policies",
+  "Target pods": "Target pods",
+  "From": "From",
+  "To ports": "To ports",
+  "NS selector": "NS selector",
+  "Namespace details": "Namespace details",
+  "Ingress rules": "Ingress rules",
+  "Pods accept all traffic by default. They can be isolated via Network Policies which specify a whitelist of ingress rules. When a Pod is selected by a Network Policy, it will reject all traffic not explicitly allowed via a Network Policy. See more details in:": "Pods accept all traffic by default. They can be isolated via Network Policies which specify a whitelist of ingress rules. When a Pod is selected by a Network Policy, it will reject all traffic not explicitly allowed via a Network Policy. See more details in:",
+  "Network Policies documentation": "Network Policies documentation",
```

Comment:
Is "Network Policies" supposed to be capitalized? It's not on line 5.

Reply:
Hi, if these are capitilization issues, pls add them to #6928! -thanks
cc @spadgett

Reply:
Thanks!